src/test_tif_loader.cpp

```diff
@@ -44,22 +44,28 @@
 
 #include "grid_map_msgs/msg/grid_map.h"
 #include "grid_map_ros/GridMapRosConverter.hpp"
+#include "geometry_msgs/msg/transform_stamped.hpp"
+#include "tf2_ros/static_transform_broadcaster.h"
 
 using namespace std::chrono_literals;
 
 class MapPublisher : public rclcpp::Node {
  public:
   MapPublisher() : Node("map_publisher") {
-    original_map_pub_ = this->create_publisher<grid_map_msgs::msg::GridMap>("elevation_map", 1);
 
     std::string file_path = this->declare_parameter("tif_path", ".");
-    std::string color_path = this->declare_parameter("tif_color_path", "");
+    std::string color_path = this->declare_parameter("tif_color_path", ".");
+    std::string frame_id = this->declare_parameter("frame_id", "map");
+    std::string topic_name = this->declare_parameter("topic_name", "elevation_map");
```

Comment:
Perhaps we could consider using remappings instead, which don't require a user to parameterize all their topic names.

Reply:
Hmm, but the target remap topic would be provided by the user?

Reply:
Yea, or more likely in the launch file.

Reply:
Done!